src/components/SiteTitle.js

```diff
@@ -3,13 +3,13 @@ import React from 'react';
 const SiteTitle = () => (
   <div className="text-center">
     <a className="no-underline inline-block py-12 md:py-8" href="/">
-      <h1 className="text-dark-pink text-6xl md:text-7xl">
-        Hacktober
-        <span className="text-light-pink text-3xl md:text-5xl uppercase block">
+      <h1 className="text-hack-logo text-6xl md:text-7xl">
+        hacktober
+        <span className="block text-hack-alt-logo text-3xl md:text-5xl">
```

Comment:
Untested but perhaps `block text-hack-alt-logo text-5xl md:text-6xl -m-8` here to sort the text size and spacing?

Reply:
Changed to `block text-hack-alt-logo text-6xl md:text-7xl -m-8` and worked as well.